src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Bulldozer started working");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Bulldozer stopped working \n");
```

Comment:
```suggestion
        System.out.println("Bulldozer stopped working");
```

Зачем нам тут перенос на новую строку?